### PR TITLE
Spectra v2.6.9

### PR DESCRIPTION
**Pull request by adi3890**
- Improvement: Post Grid - Added the option for normal pagination to disable Ajax-based pagination.
- Fix: Info Box - Removed the line-height inherit CSS to resolve an issue with extra spacing in the editor.
- Fix: Info Box - Resolved an issue in which the bottom margin for the title would not work as intended on the frontend.
- Fix: Forms - Resolved an issue in which the overall alignment would not work as intended when three forms are placed with different alignments.
- Fix: Popup Builder - Resolved an edge case issue with Astra global page layout.
- Fix: Testimonial - Resolved an issue in which the arrows would overlap the content on mobile devices.

### CODE REVIEW

File: blocks-config/post/class-uagb-post.php [hunk 1]
```diff
@@ -1826,8 +1830,11 @@ public function add_post_dynamic_script() {
 				}
 			}
 
-			if ( isset( self::$settings['grid'] ) && ! empty( self::$settings['grid'] ) ) {
+			if ( ! empty( self::$settings['grid'] ) && is_array( self::$settings['grid'] ) ) {
 				foreach ( self::$settings['grid'] as $key => $value ) {
+					if ( 'ajax' !== $value['paginationType'] ) {
```
Comment (bsf-zanev) [inline blocks-config/post/class-uagb-post.php:1835]: Should we check if `$value['paginationType']` is set as well?

File: src/blocks/post/post-grid/settings.js [hunk 2]
```diff
@@ -604,6 +605,26 @@ const Settings = ( props ) => {
 
 		return (
 			<UAGAdvancedPanelBody title={ __( 'Pagination', 'ultimate-addons-for-gutenberg' ) } initialOpen={ false }>
+				<MultiButtonsControl
```
Comment (bsf-zanev) [inline src/blocks/post/post-grid/settings.js:608]: Spacing between brackets inside this control are missing.

File: blocks-config/post/class-uagb-post.php [hunk 3]
```diff
@@ -1530,13 +1534,13 @@ public function post_pagination() {
...
+				
 				$post_attribute_array = $this->required_attribute_for_query( $attr );
-
+				
 				$query = UAGB_Helper::get_query( $post_attribute_array, 'grid' );
-
+				
 				$pagination_markup = $this->render_pagination( $query, $attr );
-
+
```
Comment (bsf-zanev) [inline blocks-config/post/class-uagb-post.php:1543]: Minor Suggestion - Can we remove these lines for cleaner code?

**Review by bsf-zanev — COMMENTED** [review]
Few suggestions.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by rajmohanreddy774 — APPROVED** [review]

File: blocks-config/post/class-uagb-post.php (hunk 1 above)
Comment (bsf-zanev) [reply blocks-config/post/class-uagb-post.php]: This has been handled in [this merge](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3301/commits/5fd604a8a9e0f318d7272795dc5f2ba4b85633da).

File: src/blocks/post/post-grid/settings.js (hunk 2 above)
Comment (bsf-zanev) [reply src/blocks/post/post-grid/settings.js:608]: This has been handled in [this merge](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3301/commits/5fd604a8a9e0f318d7272795dc5f2ba4b85633da).

File: blocks-config/post/class-uagb-post.php (hunk 3 above)
Comment (bsf-zanev) [reply blocks-config/post/class-uagb-post.php]: This has been handled in [this merge](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3301/commits/5fd604a8a9e0f318d7272795dc5f2ba4b85633da).

Comment (adi3890) [reply blocks-config/post/class-uagb-post.php]: Resolved in https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3302

File: src/blocks/post/post-grid/settings.js (hunk 2 above)
Comment (adi3890) [reply src/blocks/post/post-grid/settings.js:608]: Resolved in https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3302

File: blocks-config/post/class-uagb-post.php (hunk 1 above)
Comment (adi3890) [reply blocks-config/post/class-uagb-post.php]: Resolved in https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3302

**Review by bsf-zanev — APPROVED** [review]

**Pull request merged**